### DISCUSSION TITLE
Getting Input ID

### DISCUSSION THREAD

**Discussion by LfJohnVo** [livewire/livewire#4348] [1. Help]
I need to pass my dynamic input id from blade to my backend, this code generates input cells to appears like excel stylesheet

![image](https://user-images.githubusercontent.com/32848900/144790476-78200222-925b-4800-83b4-886b5efd83fc.png)
![image](https://user-images.githubusercontent.com/32848900/144790492-1cbc1e9b-b1fc-4bae-bd74-92b76ef4e581.png)
![image](https://user-images.githubusercontent.com/32848900/144790500-e01245f4-26c0-4873-a82c-849685d38f28.png)

**Comment by joshhanley** [comment]
@LfJohnVo any chance you can paste your code instead of screenshots?

When you paste code into github, you need to surround it in triple back ticks and then specify the file type like php or blade.

For example
\```php
\\ PHP code here
\```

\```blade
\\ HTML and blade code here
\```

Thanks!

**Reply by LfJohnVo** [reply]
Of course,

Blade:
`<div>
    <table>
        <tr>
            <td>&nbsp;</td>
            @foreach ($cabeceras as $key => $catem)
                <td ALIGN="center"><strong>{{ $catem->nombre_comercio }}</strong></td>
            @endforeach
        </tr>
        @foreach ($montos as $key => $mitem)
            <tr>
                <th ALIGN="center">$ {{ $mitem->monto }}&nbsp;&nbsp;</th>
                @foreach ($cabeceras as $key => $catem)
                    <td style="margin-left: 5%;"><input type="text" class="form-control form-control-sm"
                            placeholder="Boletos" wire:model="input" id="monto{{ $mitem->id }}-comer{{ $catem->id }}" value="boleto-monto{{ $mitem->id }}-comer{{ $catem->id }}">
                    </td>
                @endforeach
            </tr>
        @endforeach
    </table>
</div>
`

php
`<?php

namespace App\Http\Livewire;

use Livewire\Component;
use Illuminate\Support\Facades\DB;
use Illuminate\Support\Facades\Auth;

class CreateRDIComponent extends Component
{
    public $tabulador, $cabecaeras, $input;

    public function updatedInput($value)
    {
        dd($value);
    }

    public function render()
    {
        $id_est = Auth::user()->id_est;
        $tabulador = "select lt.id, lt.monto from lay_tabulador lt
        inner join tabuladores_estacionamientos ts on lt.id = ts.id_tabulador where id_estacionamiento = 8 order by lt.monto asc";
        $result1 = DB::connection('mysql2')->SELECT($tabulador);
        $cabecaeras = DB::connection('mysql2')->SELECT("
        select lc.id, lc.nombre_comercio from comercios_estacionamientos ce inner join lay_comercios lc
        on ce.id_comercio = lc.id where ce.id_estacionamiento = 8");
        return view('livewire.create-r-d-i-component', [
            'montos' => collect($result1),
            'cabeceras' => collect($cabecaeras),
            'contacabe' => count(collect($cabecaeras)),
        ]);
    }
}
`

**Reply by joshhanley** [reply]
@LfJohnVo I think something has gone wrong with the formatting and half the code is missing. Please format your code with triple backticks and the type hint like in my message above 🙂 Thanks!
